### PR TITLE
Initial Fixes

### PR DESCRIPTION
**Pull request by kevlahnota**

### CODE REVIEW

File: src/main/java/com/simpligility/maven/plugins/android/standalonemojos/ManifestMergerMojo.java [hunk 1]
```diff
@@ -372,7 +372,7 @@ public void manifestMergerV2() throws MojoExecutionException, MojoFailureExcepti
                   androidManifestFile,     // mainManifest
                   new ArrayList<File>(),   // manifestOverlays
                   manifestDependencies,    // libraries
-                  "",                      //featurename
+                  null,                      //featurename
```
Comment (kevlahnota) [inline src/main/java/com/simpligility/maven/plugins/android/standalonemojos/ManifestMergerMojo.java:375]: - https://github.com/Card-Forge/forge/pull/4977

Comment (kevlahnota) [reply src/main/java/com/simpligility/maven/plugins/android/standalonemojos/ManifestMergerMojo.java:375]: mvn clean install -DskipTests -e should work to build the jar

**Review by Hanmac — APPROVED** [review]

File: src/main/java/com/simpligility/maven/plugins/android/standalonemojos/ManifestMergerMojo.java (hunk 1 above)
Comment (Hanmac) [reply src/main/java/com/simpligility/maven/plugins/android/standalonemojos/ManifestMergerMojo.java:375]: Better Link would be to this: https://github.com/simpligility/android-maven-plugin/issues/814

**Comment by Hanmac** [comment]
@tehdiplomat i think i'm going to merge this, and then can re try to make a new release out of it

**Comment by tehdiplomat** [comment]
@Hanmac Do we need to update the snapshot script for new versions and things? Whats the plan?

**Comment by Hanmac** [comment]
@tehdiplomat the snapshot script already uses the 4.6.1 Version:
https://github.com/Card-Forge/forge/blob/038374c3c5df334e1dcae40278723302ba5aea22/.github/workflows/snapshots-android.yml#L79-L80

so no changes needed

just the release for this Repo need to be build / updated again (maybe an Action for this repo too?)

**Comment by tehdiplomat** [comment]
Ah I think @kevlahnota had built that locally and I just uploaded it last time? An action to build the repo is probably a good idea. Should we be bumping the version number to 4.6.2? So we don't clubber the functioning release? I guess pom can handle that.

**Comment by Hanmac** [comment]
yeah an Action would be cool

i'm not against bumping it

**Comment by tehdiplomat** [comment]
@kevlahnota How are you running this locally? Can we convert that into github actions here?

**Comment by kevlahnota** [comment]
> @kevlahnota How are you running this locally? Can we convert that into github actions here?

I run this
`mvn clean install -DskipTests -e`

I don't have experience using github actions so I don't know what to do with it.

**Pull request merged**